app/Http/Controllers/CollectionController.php

```diff
@@ -123,9 +122,23 @@ public function index(Request $request): Response|JsonResponse|RedirectResponse
 
     public function view(Request $request, Collection $collection): Response
     {
-        /** @var User $user */
         $user = $request->user();
 
+        if ($user === null) {
+            return Inertia::render('Collections/Index', [
```

Comment:
we show index here to have something in the background and this page is easier to fake in terms of data than an actual collection detail page